k8s/meadowlark-api-deployment.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    serviceName: meadowlark-api
+  name: meadowlark-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      serviceName: meadowlark-api
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        io.kompose.network/meadowlark-js-default: "true"
+        serviceName: meadowlark-api
+    spec:
+      containers:
+        - env:
+            - name: ALLOW_OVERPOSTING
+              value: "false"
+            - name: ALLOW_TYPE_COERCION
+              value: "true"
+            - name: ALLOW__EXT_PROPERTY
+              value: "true"
+            - name: AUTHORIZATION_STORE_PLUGIN
+              value: '@edfi/meadowlark-postgresql-backend'
+            - name: DOCUMENT_STORE_PLUGIN
+              value: '@edfi/meadowlark-postgresql-backend'
```

Comment:
Would it make sense to standardize on `'` or `"` rather than having both?

Do these value strings actually need to be delimited with one or the other? For example, docker compose doesn't need `"` around string values for the most part (maybe some exceptions though).

Reply:
That's true, this was done automatically from kompose, but according to the YAML specification can be done with double, single or no quotes. The only exception is true or false as strings to avoid converting to boolean